src/addrman.h

```diff
@@ -279,7 +279,7 @@ friend class CAddrManTest;
 #endif
 
     //! Select several addresses at once.
-    void GetAddr_(std::vector<CAddress> &vAddr, size_t max_addresses, size_t max_pct) EXCLUSIVE_LOCKS_REQUIRED(cs);
+    void GetAddr_(std::vector<CAddress> &vAddr, size_t max_addresses, size_t max_pct, Network net = Network::NET_MAX) EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
The default value can be removed since all callers of this method always provide the argument.

```suggestion
    void GetAddr_(std::vector<CAddress> &vAddr, size_t max_addresses, size_t max_pct, Network net) EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Reply:
Indeed! Done.

Reply:
Indeed! Done.